tests/performance/phrase/phrase_cases.rb

```diff
@@ -64,9 +64,8 @@ def test_phrase_cases
     (exitcode, output) = execute(node, "vespa-feed-perf < #{dirs.tmpdir}/feed-phrases.xml")
     assert_equal(0, exitcode)
     wait_for_hitcount("sddocname:foobar", 123456, 30)
-
-    node.execute('vespa-proton-cmd --local triggerFlush')
-    node.execute('vespa-proton-cmd --local triggerFlush')
+    node.trigger_flush
```

Comment:
Consider using _vespa.search["search"].first.trigger_flush_ instead. Also consider adding the die function below to nodetypes/searchnode.rb.